### ISSUE TITLE
How to run gaussian_surfels on some indoor scenes

### ISSUE THREAD

**Issue by VerseWei** [turandai/gaussian_surfels#50]
Great work! I found that Gaussian Surfels works well in object-centered scenes, but I failed to repeat the results in some indoor scenes, such as the drjohnson dataset. I don't know if I need to modify some parameters. Can you give me some advice?
![微信图片_20240919231445](https://github.com/user-attachments/assets/09e4675c-9032-4cc7-ac9b-8eb68ded3ad0)
![微信图片_20240919231445](https://github.com/user-attachments/assets/09e4675c-9032-4cc7-ac9b-8eb68ded3ad0)

**Comment by turandai** [comment]
Hi, you may try comment out [this](https://github.com/turandai/gaussian_surfels/blob/122799c2334989e0373a4f7b722f8cde30a86563/train.py#L46) which is designed for object-centric scenes. And also, for scenes with hundres of images, also consider comment out [this](https://github.com/turandai/gaussian_surfels/blob/122799c2334989e0373a4f7b722f8cde30a86563/train.py#L49) and simultaneously remove the ```prune_vis``` [here](https://github.com/turandai/gaussian_surfels/blob/122799c2334989e0373a4f7b722f8cde30a86563/scene/gaussian_model.py#L507). Hope this helps.

**Comment by VerseWei** [comment]
Thanks for your reply. I tried it according to your suggestion, but it seems that there is no improvement. In fact, I ran the scenario that you experimented in your paper. As shown below, I disabled the mask and the corresponding loss, but the result is still not ideal. Can you tell me how you did it when conducting the experiment in this scenario?
![image](https://github.com/user-attachments/assets/f21cb663-3b80-4aad-adbb-86946d4c63c8)
![image](https://github.com/user-attachments/assets/1986c76f-a164-47f2-91a8-787173875392)
@turandai

**Comment by turandai** [comment]
Hi, I tried also turn off the [loss_opac](https://github.com/turandai/gaussian_surfels/blob/122799c2334989e0373a4f7b722f8cde30a86563/train.py#L142), resulting in similar performance in the paper. I am sorry there are inconsistencies in hyper-params for scenes with different scales. It also took me some time to reproduce it. 
More details may help: 1) the images shown in the paper should be the IMG_6302. 3) when extract mesh for larger scale scenes, you might also increase the resolution of occupancy filtering grid [here](https://github.com/turandai/gaussian_surfels/blob/122799c2334989e0373a4f7b722f8cde30a86563/render.py#L33) and the poisson depth.
![IMG_6302](https://github.com/user-attachments/assets/b7dbd07e-4aaa-476b-bfa0-30290787a549)
